apps/token-manager/app/src/App.js

```diff
@@ -124,134 +111,66 @@ class App extends React.Component {
-                              {tokenSymbol && (
-                                <Badge.App>{tokenSymbol}</Badge.App>
-                              )}
-                            </Title>
-                            <AssignTokensButton
-                              title="Assign Tokens"
-                              onClick={this.handleLaunchAssignTokensNoHolder}
-                            />
-                          </AppBarContainer>
-                        </AppBar>
-                      }
-                    >
-                      {appStateReady && holders.length > 0 ? (
-                        <Holders
-                          holders={holders}
-                          groupMode={groupMode}
-                          maxAccountTokens={maxAccountTokens}
-                          tokenAddress={tokenAddress}
-                          tokenDecimalsBase={tokenDecimalsBase}
-                          tokenName={tokenName}
-                          tokenSupply={tokenSupply}
-                          tokenSymbol={tokenSymbol}
-                          tokenTransfersEnabled={tokenTransfersEnabled}
-                          userAccount={userAccount}
-                          onAssignTokens={this.handleLaunchAssignTokens}
-                          onRemoveTokens={this.handleLaunchRemoveTokens}
-                        />
-                      ) : (
-                        <EmptyState
-                          onActivate={this.handleLaunchAssignTokensNoHolder}
-                        />
-                      )}
-                    </AppView>
-                  )}
-                </WindowSize>
-                <SidePanel
-                  title={
-                    assignTokensConfig.mode === 'assign'
-                      ? 'Assign tokens'
-                      : 'Remove tokens'
-                  }
-                  opened={sidepanelOpened}
-                  onClose={this.handleSidepanelClose}
-                  onTransitionEnd={this.handleSidepanelTransitionEnd}
-                >
-                  {appStateReady && (
-                    <AssignVotePanelContent
-                      opened={sidepanelOpened}
-                      tokenDecimals={numData.tokenDecimals}
-                      tokenDecimalsBase={tokenDecimalsBase}
-                      onUpdateTokens={this.handleUpdateTokens}
-                      getHolderBalance={this.getHolderBalance}
-                      maxAccountTokens={maxAccountTokens}
-                      {...assignTokensConfig}
-                    />
-                  )}
-                </SidePanel>
-              </Main>
-            </PublicUrl.Provider>
-          </ToastHub>
-        </WindowSizeProvider>
-      </Root.Provider>
+      <div css="min-width: 320px">
+        <Main>
+          <AppLayout
+            title="Token Manager"
+            afterTitle={tokenSymbol && <Badge.App>{tokenSymbol}</Badge.App>}
```

Comment:
Maybe `title` accepts an element/node instead of using _after_?

Reply:
Because `title` need to be wrapped separately: https://github.com/aragon/aragon-apps/pull/666/files#diff-2e20014b3077a7461c11aabfbe2c53abR27